### PR TITLE
Movies

### PR DESCRIPTION
**Pull request by CJNewcomer**
Here is the movie profile page - please review

### CODE REVIEW

File: public/stylesheets/movie-profile.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+   font-weight: normal;
+   line-height: 21px;
+}
+
+p {
+    text-align: left;
+    word-wrap: normal;
+    content: normal;
+}
+
```
Comment (genagain) [inline public/stylesheets/movie-profile.css]: Please remove these extra lines.

File: public/stylesheets/movie-profile.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    border-radius: 5px;
+}
+
+.image-column {
+    flex-direction: column;
+    justify-content: left;
+    width: 180px;
+    height: 100%;
+    text-align: center;
+    margin: 5.5px 20px 10px 0
```
Comment (genagain) [inline public/stylesheets/movie-profile.css]: Please add a semicolon here.

File: routes/movies.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+const csrfProtection = csrf({ cookie: true });
+
+const db = require('../db/models');
+
+const asyncHandler = (handler) => (req, res, next) => handler(req, res, next).catch(next);
+
+router.get('/:id(\\d+)', asyncHandler(async(req, res) => {
+    const movieId = parseInt(req.params.id, 10);
+    const movie = await db.Movie.findByPk(movieId);
+    // const year = movie.releaseDate.toString().split(' ')[3];
```
Comment (genagain) [inline routes/movies.js]: Please remove this commented out code.

File: routes/movies.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    const year = movie.releaseDate.getFullYear();
+    res.render('movie-profile', { title: 'Movie Profile', movie, year });
+}));
+
+// TODO: Status Route
+    // add button with drop down functionality (pug,css)
```
Comment (genagain) [inline routes/movies.js]: If you plan on leaving in this TODO, please make sure you reference a Github issue for implementing the feature that needs the Status route.

File: views/movie-profile.pug [hunk 5]
```diff
@@ -0,0 +1,29 @@
+doctype html
+html
+    head
+        meta(charset="UTF-8")
+        meta(name="viewport", content="width=device-width, initial-scale=1.0")
+        link(rel="stylesheet", href="/stylesheets/movie-profile.css")
+        title Movie-Profile
+    body
```
Comment (genagain) [inline views/movie-profile.pug]: Can you please tell me why this isn't extending the `layout.pug` file?

**Review by genagain — CHANGES_REQUESTED** [review]

File: public/stylesheets/movie-profile.css (hunk 2 above)
Comment (CJNewcomer) [reply public/stylesheets/movie-profile.css]: Added.

File: public/stylesheets/movie-profile.css (hunk 1 above)
Comment (CJNewcomer) [reply public/stylesheets/movie-profile.css]: Removed.

File: routes/movies.js (hunk 4 above)
Comment (CJNewcomer) [reply routes/movies.js]: Removed.

**Review by CJNewcomer — COMMENTED** [review]
Ready for another review, please.

**Review by genagain — APPROVED** [review]
Looks good to me! Ship it 🚀

**Pull request merged**